direct/nn/vsharp/vsharp.py

```diff
@@ -147,8 +148,8 @@ class VSharpNet(nn.Module):
 
     def __init__(
         self,
-        forward_operator: Callable,
-        backward_operator: Callable,
+        forward_operator: callable,
+        backward_operator: callable,
         num_steps: int,
```

Comment:
That's interesting. but I think it can be more complete, you can define an input type and a return type.

Reply:
Number of input(s) may vary. Also in other modules this is the way we define it.

Reply:
Done as `Callable[[tuple[Any, ...]], torch.Tensor]`